private/dir.go

```diff
@@ -0,0 +1,55 @@
+package private
```

Comment:
As a non native english speaker, I find `private` to be too generic and I've honestly not fully understood the purpose of this package other than path sanitization. Could you either explain why this `private` name or add a package comment to explain?